AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.ts

```diff
@@ -309,6 +319,20 @@ export class SummaryComponent implements OnInit, OnDestroy {
         }
     }
 
+    async oberserableTimer(hearingId: string): Promise<UpdateBookingStatusResponse> {
```

Comment:
could we name this function something like `getHearingStatus`?

Reply:
name changed